apps/src/StudioApp.js

```diff
@@ -282,7 +282,7 @@ StudioApp.prototype.configureRedux = function (reducers) {
  */
 StudioApp.prototype.createReduxStore_ = function () {
   var combined = combineReducers(_.assign({}, commonReducers, this.reducers_));
-  this.reduxStore = redux.createStore(combined);
+  this.reduxStore = redux.createStore(combined, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
```

Comment:
Ah nice catch, will remove for this PR. It's a pretty neat devtool extension:

![image](https://cloud.githubusercontent.com/assets/206973/19947943/b898ffd2-a120-11e6-851b-744efe591cb9.png)